src/app/components/footer/footer.component.scss

```diff
@@ -3,16 +3,14 @@
 .footer {
   color: #757982;
   align-items: center;
-
   font-weight: normal;
   line-height: 30px;
   font-size: 12px;
-
   font-family: Roboto, Helvetica Neue, Arial, sans-serif, Apple Color Emoji,
     Segoe UI Emoji, Segoe UI Symbol;
   text-decoration: none;
-
-  border-top: 2px solid #1a191a !important;
+  background-color: transparent;
```

Comment:
Should it to be transparent?

Reply:
yes. in app css footer has style and there i assign the color.or i can give the color here